### ISSUE TITLE
https://tldr.ostera.io/opensearch.xml is missing an icon

### ISSUE THREAD

**Issue by FreifunkerEZ** [tldr-pages/tldr#1404]
I really like the web client and I love that it provides an open-search-file to be included in browsers.

Unfortunately there is no icon defined in there, so now, while it IS in my list of search-engines in Firefox, the button cannot be distinguished between the empty one next to it. Which makes it seem like it is not there.
![image](https://user-images.githubusercontent.com/17656471/26917641-5e9a37e8-4c2f-11e7-9c6d-8a988cd9e981.png)
That box below the purple Y (Yodel?) is the tldr-web-client :(
![image](https://user-images.githubusercontent.com/17656471/26917714-9fb38ac2-4c2f-11e7-8958-903580bed604.png)

**Comment by waldyrious** [comment]
Pinging @ostera

**Comment by sbrl** [comment]
tldr-pages could do with an official icon!

**Comment by waldyrious** [comment]
@sbrl what do you mean? The website at https://tldr-pages.github.io / https://tldr.sh already has a favicon, which is the same image that shows up at https://github.com/tldr-pages.

Am I missing something,or are you? :smile:

**Comment by waldyrious** [comment]
In any case, this is an issue for [ostera/tldr.jsx](https://github.com/ostera/tldr.jsx); there's nothing we can do about it in this repo. Can you please report the issue [there](https://github.com/ostera/tldr.jsx/issues), @FreifunkerEZ? Thanks :)

**Comment by sbrl** [comment]
HUh! Must have been some weirdness on my end, @waldyrious :P
